### ISSUE TITLE
Upgrade to tiledb v2.1

### ISSUE THREAD

**Issue by Hoeze** [TileDB-Inc/TileDB-Py#393]
Hi, would it be possible to upgrade this to the new TileDB version for improved threading?
```
#> mamba install "tiledb>=2.1"
Looking for: ["tiledb[version='>=2.1']"]

Encountered problems while solving.
Problem: package tiledb-py-0.6.6-py37haae2426_0 requires tiledb >=2.0.8,<2.1.0a0, but none of the providers can be installed
```

**Comment by ihnorton** [comment]
Yes, we will have a version bump available by tomorrow.

**Comment by ihnorton** [comment]
Apologies for the delay here: 0.7 is now available with TileDB Embedded (libtiledb) 2.1.2 on [conda](https://anaconda.org/conda-forge/tiledb-py) and [pypi](https://pypi.org/project/tiledb/).

**Comment by Hoeze** [comment]
Thanks a lot @ihnorton!
